lib/features/auth/repositories/auth_repository.dart

```diff
@@ -107,6 +108,14 @@ class AuthRepositoyImpl with AuthRepositoy {
         UserExt.tag,
         jsonEncode(newUserExt.toJson()),
       );
+
```

Comment:
this code looks like duplicate of the ` Future<UserExt> fetchUserExtension(String guid) async {` below.
Can we reuse it in this place?